tasks/configs.yml

```diff
@@ -3,26 +3,26 @@
 - name: configs > {{ __manala_php_sapi|capitalize }} - Templates
   template:
     src:  "{{ item.template|default(manala_php_configs_template|ternary(manala_php_configs_template, 'configs/empty.j2')) }}"
-    dest: "{{ manala_php_configs_dir_base }}/{{ __manala_php_sapi }}/conf.d/{{ item.file }}"
+    dest: "{{ manala_php_versions[manala_php_version|string]['config_dir'] }}/{{ __manala_php_sapi }}/conf.d/{{ item.file }}"
   notify:
-    - php {{ __manala_php_sapi }} restart
-  with_items: "{{ manala_php_configs + vars['manala_php_' + __manala_php_sapi + '_configs'] }}"
+    - php fpm restart
```

Comment:
Interesting point. I've digged a little, and fpm is the *only* sapi having an attached service. Regarding the fact that we can not (or do we ?) select what to notify, i notify onnly fpm, and check is fpm has been selected in the handler itself.
Another approach could be just to notify an agnostic "php restart", and check fpm in the handler too. Would you prefer ?